postgres/init/20-portal.sql

```diff
@@ -113,6 +119,15 @@ CREATE TABLE portal.company_role_descriptions (
 );
 
 
+CREATE TABLE portal.company_user_role_descriptions (
+    company_user_role_id uuid,
+    language_short_name character(2),
+    description character varying(255) NOT NULL,
+    CONSTRAINT pk_company_user_role_descriptions PRIMARY KEY (company_user_role_id, language_short_name),
+    CONSTRAINT fk_company_user_role_descriptions_company_role_id FOREIGN KEY (company_user_role_id) REFERENCES portal.company_user_roles(id),
+    CONSTRAINT fk_company_user_role_descriptions_language_short_name FOREIGN KEY (language_short_name) REFERENCES portal.languages(short_name)
+);
+
```

Comment:
"user_role_descriptions" instead of "user_role_descriptions" ?

Reply:
you mean 'user_role_descriptions' instead of 'company_user_role_descriptions'? ok, will do the rename